build/windows/installer/Installer.wixproj

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" Sdk="WixToolset.Sdk/4.0.0-rc.2">
+    <PropertyGroup>
+        <DebugType>none</DebugType>
+        <OutputName>installer</OutputName>
+    </PropertyGroup>
+    <ItemGroup>
+        <PackageReference Include="WixToolset.UI.wixext" Version="4.0.0-rc.2" />
+        <PackageReference Include="WixToolset.Heat" Version="4.0.0-rc.2" />
```

Comment:
What are these dependencies used for?

Reply:
The WixToolset is split into two packages:
`WixToolset.UI.wixext` is for the installer UI
`WixToolset.Heat` is for the harvesting functionality which aggregates all files in the `assets` and `credits` dirs